### DISCUSSION TITLE
/posts/why_am_i_blogging/

### DISCUSSION THREAD

**Discussion by rjachuthan** [rjachuthan/riturajwrites#2] [Announcements]
This is the post where I have explained about why I have started to blog.

**Comment by supratim1121992** [comment]
Good start. I feel you're undermining yourself. This is very well written. Almost has a poetic feel to it. Coding in poetry... Now that's something you should explore 😄

**Reply by rjachuthan** [reply]
Thank you for your kind words and encouragement! Appreciate your feedback. I'll definitely consider exploring the idea of coding in poetry, it sounds intriguing and challenging, but also fun! 😄
